backend/tests/test_db.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import pytest
+
+from backend.models import User, Participant, Hacknight
+
+
+def test_add_new_user_to_db(_db, new_user):
+    """
+    GIVEN mocked db session, and user data
+    WHEN a new user added to db
+    THEN check the user data from db
```

Comment:
Gherkin syntax is descriptive and used for the BDD approach. I would stick with classic docstring convention. Though we can discuss this matter.